src/foam/u2/wizard/agents/AnalyticEventsAgent.js

```diff
@@ -83,6 +82,16 @@ foam.CLASS({
           extra: `${this.window.innerWidth}x${this.window.innerHeight}`
         });
       }
+    },
+
+    async function logAnalyticsEvent(evt) {
```

Comment:
Should this be a listener?

Reply:
Why a listener?

Reply:
Can you remove the async and await?

Reply:
async and await is to allow the caller to call it with await https://github.com/nanoPayinc/NANOPAY/pull/16466/files.

To remove the async and await the function need to add `return` on the dao.put() line.

Reply:
Arent all props in the function always supposed to be called linked to the contextAgent's `this`? Unless exported functions are auto bound to their class?

Reply:
export magic https://github.com/kgrgreer/foam3/blob/9bea9591764cd057cb93c70c63eae99b4f7a94c1/src/foam/core/Method.js#L216-L217